app/controllers/vendor_api/error_handling.rb

```diff
@@ -0,0 +1,32 @@
+require 'active_support/concern'
+
+module VendorApi::ErrorHandling
```

Comment:
Non-blocking: since this is only used in the base controller, do we need a concern?